pkg/firmware/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Make modules from a recentish kernel available
+FROM linuxkit/kernel:4.14.12 AS kernel
+
+FROM linuxkit/alpine:34518265c6cb63ff02074549cc5b64bef40c336f AS build
+RUN apk add --no-cache git kmod
+
+# Clone the firmware repository
+ENV FW_URL=git://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git
+ENV FW_COMMIT=65b1c68c63f974d72610db38dfae49861117cae2
```

Comment:
You could do `FROM liunxkit/firmware-all AS firmware-all` then use `COPY --from` instead of cloning again, would make it easier to keep them in sync (or would it?).

Reply:
yeah, but then we'd need to update the hashes as well...either way it's two updates but if we were to re-use `liunxkit/firmware-all` we also have a dependency between packages which is more annoying to handle

Reply:
ok. Might be worth a comment above both FW_COMMIT definitions saying "remember to update the other one"?